src/main/java/com/softserveinc/ita/pageobjects/CategoriesPage.java

```diff
@@ -0,0 +1,17 @@
+package com.softserveinc.ita.pageobjects;
+
+import static com.codeborne.selenide.Selenide.$x;
+
+public class CategoriesPage extends BasePage<CategoriesPage>{
+
+    private final String subCategorySelector = "(//a[contains(@class , 'tile-cats__heading')])[1]";
```

Comment:
need to review xpath's
I will do it later and suggest something

Reply:
if I understand, it always will be the first subcategory? probably better to specify or through random chose category